starwars/src/App.js

```diff
@@ -28,14 +31,20 @@ class App extends Component {
         throw new Error(err);
       });
   };
-
-  render() {
-    return (
+   render() {
+    // console.log(this.state)
+    console.log()
+     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
-      </div>
-    );
-  }
+         <CharList chars = {this.state.starwarsChars}
+               cars = {this.state.starwarsChars.vehicles}/>
+                       console.log(this.state.starwarsChars.vehicles)
```

Comment:
Careful leaving console logs in your code. This one actually rendered to the dom because it is in JSX and not JS.